ephios/core/forms/users.py

```diff
@@ -200,14 +218,22 @@ def save(self, commit=True):
         return group
 
 
-class UserProfileForm(ModelForm):
+class UserProfileForm(PermissionFormMixin, ModelForm):
     groups = ModelMultipleChoiceField(
         label=_("Groups"),
         queryset=Group.objects.all(),
         widget=Select2MultipleWidget,
         required=False,
         disabled=True,  # explicitly enable for users with `change_group` permission
     )
+    is_staff = PermissionField(
```

Comment:
does this still set the actual is_staff flag? couldn't find that in the code

Reply:
well `PermissionField` is just a BooleanField on a ModelForm so it does set the flag, yes

Reply:
could rename it to `PermissionBooleanField` to make that more clear as well as emphasize the it's either one or off, not a select.

Reply:
> well `PermissionField` is just a BooleanField on a ModelForm so it does set the flag, yes

ah well. we just didn't use it that way before, because it only operated as an additional field that doesn't exist on the model itself. just a bit intransparent that it updates the flag as well as the permissions

Reply:
what docstring? of `PermissionField` or the Form I use it in?

Reply:
PermissionField